docs/.vitepress/vitepress/components/dev/A11yTag.vue

```diff
@@ -0,0 +1,13 @@
+<template>
```

Comment:
For the file name, please follow the convention with **kebab-case** instead of PascalCase.